.github/workflows/build_and_test/action.yml

```diff
@@ -46,3 +46,13 @@ runs:
         name: golden-test-failures
         path: test/golden/failures
         if-no-files-found: ignore
+
+    - name: 💾 Cache Gradle
+      uses: actions/cache@v4
+      with:
+        key: gradle
+        path: /home/runner/.gradle
+
+    - name: 🔨 Build Apk
+      shell: bash
+      run: flutter build apk --debug
```

Comment:
Let's go one step further and upload it as an artifact

Reply:
Ah yeah, good idea 👍